query-advisor/where-to-find.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Where to Find Query Advisor'
+backlink_href: /docs/query-advisor
+backlink_title: 'Query Advisor'
+---
+
+Once Query Advisor is configured and collecting EXPLAIN plans, optimization insights appear in several locations throughout pganalyze:
```

Comment:
(feel free to disregard) I don't have good alternatives, but this sounds like there is some special configuration is needed for Query Advisor. In Getting Started, it says "Query Advisor requires the pganalyze collector to be properly configured for query monitoring", which is more accurate - although I'm not sure putting all of these sentences here makes sense 🤔